pkgs/development/python-modules/resolvelib/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "resolvelib";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "sarugaku";
+    repo = "resolvelib";
+    rev = version;
+    sha256 = "0r7cxwrfvpqz4kd7pdf8fsynzlmi6c754jd5hzd6vssc1zlyvvhx";
+  };
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  disabledTestPaths = [
+    # missing dependency commentjson
+    "tests/functional/cocoapods/test_resolvers_cocoapods.py"
```

Comment:
You can cherry-pick https://github.com/SuperSandro2000/nixpkgs/commit/415e2b44c6f9fe885d0a83cd683833dcce20f213 from me.